public/locales/en/lobby.json

```diff
@@ -14,7 +14,11 @@
       "full": "Full",
       "join": "Join",
       "play": "Play",
-      "your_game": "Your game"
+      "your_game": "Your game",
```

Comment:
In this context isn't it a "match" rather than a "game"?

Reply:
Yes, you're right 👍